bin/oref0-dexusb-cgm-loop.py

```diff
@@ -34,7 +34,7 @@
 # @reboot cd /home/pi/openapsdir && python -u /usr/local/bin/oref0-dexusb-cgm-loop >> /var/log/openaps/cgm-loop2.log
 # this will start this python script at reboot, and log to /var/log/openaps/cgm-loop2.log
 #
-# Step 3. Disable the default cgm loop in crontab, because this script will invoke openaps get-bg
+# Step 3. Disable the default cgm loop in crontab, because this script will invoke oref0-get-bg
```

Comment:
this is a script I used in the 0.5.0 days. With 0.7.0 the G4-Go script is much better alternative. Please don't make changes to this file. I work on a PR to delete it. Feel free to delete in your PR as well.